src/components/sections/FAQSection.astro

```diff
@@ -76,10 +76,10 @@ import ListItem from "../common/ListItem.astro";
             <BlockListBody>
                 <List>
                     <ListItem>
-                        There are a number of ways to get to the convention centery by car.
+                        There are a number of ways to get to the convention center by car.
                     </ListItem>
                     <ListItem>
-                        See the Visit Pasadena site linked above for the most accurate information.
+                        See the <a href="https://www.visitpasadena.com/convention-center/attendees/directions-parking/">Visit Pasadena</a> site for the most accurate information.
```

Comment:
my subjective opinion but I would find it easier to just click the link in this section rather than scan up the page to find the link